src/index.ts

```diff
@@ -1,89 +1,148 @@
-import { Kafka, logLevel } from "kafkajs";
-import { Entry, FhirMapping, FhirPlugin, Table } from "./types";
-import { GetFhirPlugins, GetTableMappings, ValidateFhirMappingsJson } from "./util";
-import { loadDataIntoClickhouse, selectByIdClickhouse, alterRowIntoClickhouse } from "./clickhouse/utils";
-
-const fhirMappings: FhirMapping[] = require("./data/fhir-mapping.json");
-const fhirMappingValidationErrors = ValidateFhirMappingsJson(fhirMappings);
-if (fhirMappingValidationErrors.length > 0) {
-  console.error("Invalid fhir-mapping.json file");
-  fhirMappingValidationErrors.forEach((error) => console.error(error));
-  process.exit(1);
-}
+import {Kafka, logLevel} from 'kafkajs'
+import {
+  registerMediator,
+  activateHeartbeat,
+  fetchConfig
+} from 'openhim-mediator-utils'
+import {promisify} from 'util'
+import {Entry, FhirMapping, FhirPlugin, Table} from './types'
+import {
+  GetFhirPlugins,
+  GetTableMappings,
+  ValidateFhirMappingsJson
+} from './util'
+import {
+  loadDataIntoClickhouse,
+  selectByIdClickhouse,
+  alterRowIntoClickhouse
+} from './clickhouse/utils'
+import mediatorConfig from './data/mediator-config.json'
+
+let fhirMappings: FhirMapping[] = require('./data/fhir-mapping.json')
+// Set default mediator FHIR mapping from config file
+mediatorConfig.config.fhirMappings = JSON.stringify(fhirMappings, null, 2)
 
-const plugins: Map<string, FhirPlugin> = GetFhirPlugins(fhirMappings);
+const plugins: Map<string, FhirPlugin> = GetFhirPlugins(fhirMappings)
```

Comment:
```suggestion
const plugins: Map<string, FhirPlugin> = getFhirPlugins(fhirMappings)
```

Reply:
This was an existing function. I also don't like function starting with captal unless they are in classes but this occurs a few times in this repo and I think should be corrected separately.

Reply:
I decided to resolve this anyways.